stylus/components/button.styl

```diff
@@ -99,11 +99,8 @@ $button
             cursor not-allowed
 
     &[aria-busy=true]
-        > span::after
-            @extend    $icon-16
-            @extend    $icon-spinner-white
-            position   relative
-            top        rem(-1)
+        opacity .5
+        pointer-events none
```

Comment:
Why are we adding these 2 rules ?

Reply:
For this:
> - apply disabled styles to busy button
> - prevent click on busy button

Reply:
It was some of the request from the designer about busy buttons

Reply:
Ah yes ok. Wouldn't it be possible to set a `disabled` attribute on the button when it's busy ?

Reply:
I don't know. I asked myself the question but I figured that since there are two distinct aria-attributes for those two, it may not be a good idea, semantically speaking.
But I actually don't know enough on the aria topic to be sure, that's just a feeling I have.

Reply:
Mmh alright. I'm worried about pointer-events as it effectively "kills" the element through css — makes it hard to figure out why events are not firing for a certain element. Would changing the cursor be a viable option, or would we miss something ?

Reply:
The thing is that a button can be disabled but a link can't and the Button style can apply to either a button or a link. 
The `pointer-events` trick is the only one afaik that can't prevent clicks on both elements. 
Do you see any concrete usecase that could be a problem?

Reply:
Yeah I understand why disabling it is not a good fit. My concern is something like this https://codesandbox.io/s/xl4w5j8y14 — but I don't know if we ever use stuff like that.

If we want to make the button look inert, I think `cursor: auto` would be enough? If we really want to prevent all clicks on busy buttons… I think js is the right place to do that, not a css rule. Unless `pointer-events` does achieves something that `cursor` doesn't?

Reply:
> My concern is something like this https://codesandbox.io/s/xl4w5j8y14 — but I don't know if we ever use stuff like that.

I don't get what I'm supposed to see here. :/

Also `cursor` doesn't prevent anything, it just change the cursor, you can still click the link/button. 
But I see your point about not really being CSS's job to deal with this behaviour.

I'm ok removing this property but what do we do for links then? Directly dealing with this in the component ?

Reply:
Discussed it live; the choices are:

1. Only visually disable the button through css, actual clicks have to be disabled via js
2. Disable everything via css, but if we ever need interactions on a busy button we'll have to change this

There's no use case for 2 at the moment, so we'll keep it as is in the PR and update this part of the code if we ever need to.